flocx_market/objects/offer_contract_relationship.py

```diff
@@ -16,16 +16,13 @@ class OfferContractRelationship(base.FLOCXMarketObject):
     }
 
     @classmethod
-    def get(cls, marketplace_offer_id, contract_id):
-        if (contract_id is None) and (marketplace_offer_id is None):
+    def get(cls, marketplace_offer_id, contract_id, status):
```

Comment:
Ah, I think this relationship has a uuid, right? I think we should have the `get` method just return based on the uuid; and then create a `get_all` that gets a list. And instead of taking in each possible filter parameter, do something like: https://github.com/openstack/ironic/blob/master/ironic/objects/node.py#L303